### PR TITLE
Debugging windows wheels

### PR DESCRIPTION
**Pull request by jcrist**
They build fine, but are failing to upload for some reason.

### CODE REVIEW

**Comment by jcrist** [comment]
So here was the issue:

- PyPI has a validation check that the `long_description` in a built wheel is valid.
- For restructured text, this means checking that the length of each header matches the length of the matching underline:
```
This is ok
----------

This is invalid
---------
```

- I have an emoji in one of the headers "🥒". This is not a single byte character.
- The windows box was defaulting to non utf-8 (I think utf-16?) when reading in the readme before writing it to the built wheel, which changed the bytes. This led pypi to invalidate what was a valid readme on all other platforms.
- Fixing the `setup.py` to ensure `utf-8` was used to load the readme fixed things.

**TL;DR** -  unicode is hard, windows is dumb, computers are dumb

computers are dumb
